front/app/containers/Admin/index.tsx

```diff
@@ -96,8 +96,10 @@ const AdminPage = memo<Props & WithRouterProps>(
     const [adminFullWidth, setAdminFullWidth] = useState(false);
     const [adminNoPadding, setAdminNoPadding] = useState(false);
 
-    const userCanViewAdmin = usePermission({
-      item: { type: 'route', path: '/admin' },
+    const userCanViewPath = usePermission({
+      // If we're in this component, we're sure
+      // that the path is an admin path
+      item: { type: 'route', path: pathname },
       action: 'access',
     });
```

Comment:
Do we actually need this? The check in front/app/containers/Admin/routes.tsx already should do the same, right?

Reply:
Indeed, everything seems to work even without it.